### DISCUSSION TITLE
Linux Support

### DISCUSSION THREAD

**Discussion by YassinEldeeb** [YassinEldeeb/deadliner#9] [Q&A]
### Please share with me your feedback when running Deadliner on Linux so that we can make it work perfectly for Linux users.

**Comment by zicklag** [comment]
Tried it out real quick and worked so far. This is a neat project and I'll let you know what happens! :heart: 

The only thing I notice so far is that every time I click the save button, it starts a new scheduler that I can see in the tray menu.

**Reply by YassinEldeeb** [reply]
Thanks! really appreciate your support 😊
Yeah, I'm not really sure about the cause of this issue, maybe the communication between the GUI and the scheduler is broken on Linux(the scheduler starts an HTTP server and the GUI hits an endpoint on that server to trigger a shutdown), I'll dive into it. Again thanks for your feedback 🙏

**Reply by YassinEldeeb** [reply]
I'm opening an issue right now with this.
